### ISSUE TITLE
Move the delete endpoint for Rois to the Roi type and deprecate delete_rois_from_account

### ISSUE THREAD

**Issue by teije01** [vandersat/vds-api-client#26]
Currently, deleting Rois us done using:

```python
from vds_api_client import VdsApiV2

vds = VdsApiV2()
vds.delete_rois_from_account(vds.rois.filter(description_regex='Selection to Delete'))
```

However, now we can deprecate the `delete_rois_from_account` in favor of a new delete method on an Roi type:
```python
roi = vds.rois[25000]
roi.delete()
```

**Comment by chookity-pokk** [comment]
I just looked through the repo and could only find the `vds.delete_rois_from_account` in the readme. Was this already fixed?

**Comment by teije01** [comment]
Nope, issue is still open and no one started on this yet. Functionality is still as stated in the readme.

**Comment by chookity-pokk** [comment]
Okay, so then what scripts need to be edited? Because I looked around and didn't happen to find it which is why I asked.

**Comment by teije01** [comment]
`delete_rois_from_account()` is a method of `VdsApiV2` which is a class defined in `vds_api_client/api_v2.py`. This method requires a deprecation warning while we implement the `delete` method in the `Roi` class located in `vds_api_client/types.py`.

**Comment by teije01** [comment]
Dear @ragreenburg , I realize you might have looked at this as I attached the label 'good first issue'. I might have been misplaced here, as I think this requires a bit of knowledge on this package. If you are still eager to try, you are welcome, and I will help you through the PR process

**Comment by chookity-pokk** [comment]
Sure, I would like to move forward with a bit of help if you don't mind.

**Comment by teije01** [comment]
The tricky thing is that it is hard to test your code if you don't have api access (which I guess you don't or do you?). Because of that, you cannot exactly test your code once you've written it. Nevertheless, the steps to take are as follows:

If you are purely interested in going through the pull request sequence, please only do step 1 and 2. 

Steps to take:
1. You'll need to [fork](https://www.earthdatascience.org/workshops/intro-version-control-git/about-forks/) the repository, make changes there and create a pull request to the master branch on this repository
2. add a deprecation warning to the `delete_rois_from_account()` method in VdsApiV2 which is a class defined in vds_api_client/api_v2.py stating that this method will be dropped after beginning of 2021
3. implement `delete` method on `Roi` Roi class located in vds_api_client/types.py similar to other endpoints (and with the same functionality).

After that tests need to be written and the documentation needs to be adapted, but it's probably best if I take on that part. I can understand if you rather work on a package that could be useful to you.

**Comment by chookity-pokk** [comment]
I am only seeing `delete_rois_from_account()` in `vds_api_client/vds_api_base.py` under the `VdsApiBase` class.

**Comment by teije01** [comment]
you are right, it is inherited from there!

**Comment by chookity-pokk** [comment]
So I added the depreciation warning and made a function inside of the `Roi` class named `delete` but I am a bit stuck on what the logic should be inside of that function.

**Comment by teije01** [comment]
In that case, leave that point to me. Can you open a pull request with the deprecation warning? I will tag you on the remaining work.

**Comment by teije01** [comment]
closing as the API went offline today
